doc/radosgw/adminops.rst

```diff
@@ -1642,6 +1642,85 @@ Special Error Responses
 :Description: Unable to link bucket to specified user.
 :Code: 409 Conflict
 
+Change Bucket's Owner
```

Comment:
Suggest `Change the Owner of a Bucket`
In part because the apostrophe might complicate future linking.  Yes this is minor.